image_transport/include/image_transport/simple_publisher_plugin.hpp

```diff
@@ -189,13 +203,13 @@ class SimplePublisherPlugin : public PublisherPlugin
 private:
   struct SimplePublisherPluginImpl
   {
-    explicit SimplePublisherPluginImpl(rclcpp::Node * node)
-    : node_(node),
-      logger_(node->get_logger())
+    explicit  SimplePublisherPluginImpl(RequiredInterfaces required_interfaces)
```

Comment:
```suggestion
    explicit SimplePublisherPluginImpl(RequiredInterfaces required_interfaces)
```
Removes extra space.